mne_connectivity/tests/test_effective.py

```diff
@@ -39,3 +39,40 @@ def test_psi():
 
     assert np.all(conn_cwt.get_data() > 0)
     assert conn_cwt.shape[-1] == n_times
+
+
+def test_psi_time():
+    """Test Phase Slope Index (PSI) estimation across time."""
```

Comment:
From a quick glance the test looks good to me, let's see if it passes once the `verbose` param is added to the new function.